app/app/static/custom/ourscript.js

```diff
@@ -0,0 +1,117 @@
+var timers = [];
```

Comment:
can you put a comment on top of this file... like
```
/*
This is where we will put all of our JavaScript functions.
*/
```

to be honest, i dont really care about splitting the JavaScript for this project. It is much more complex (if you want to have a look, search for webpack and module bundlers). In modern Javascript frameworks, the bundling is automatically handled such that it makes it so easy to import one module on another... unfortunately not in this case

Reply:
don't use `var`, use let or const 
see https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var